### PR TITLE
161780564 User can create article

### PR DESCRIPTION
**Pull request by innocentEdosa**
#### Description
This PR implements the functionality for creating articles.
it creates a function which creates slugs from each article

#### Type of change
- feature
  - created slugGenerator function
  - created /articles endpoint
  - wrote test for article endpoint
  - wrote validation for request body

#### How Has This Been Tested?
clone repo
checkout to feature/#161780564/-user-create-article branch
run npm start
test with postman

#### Pivotal tracker id
#161780564

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 language: node_js
 node_js:
   - "stable"
+addons:
+  postgresql: "9.4"
 services:
   - postgresql
+before_script:
+- psql -c "create database authorsHaven_test;" -U postgres
 after_success: npm run coverage
```
Comment (kayroy247) [inline .travis.yml]: You need to add a new line after line 10.

**Comment by Bobjayafam** [comment]
LGTM

**Review by kayroy247 — COMMENTED** [review]
LGTM

File: .travis.yml (hunk 1 above)
Comment (innocentEdosa) [reply .travis.yml]: thanks kay
already implemented changes 
thanks

**Pull request merged**